api-tests/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "api-tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "api-tests": "vitest --config vitest.config.ts $VERSION",
```

```suggestion
    "test": "vitest",
```
What is `$VERSION`? and do you need it?